qemu/tests/cfg/uefi_check_gui_config.cfg

```diff
@@ -0,0 +1,19 @@
+- uefi_check_gui_config:
+    only q35
+    only ovmf
+    only Linux
+    type = uefi_check_gui_config
+    timeout = 360
+    boot_menu = on
+    enable_sga = yes
+    boot_menu_hint = "Boot Options"
+    boot_menu_key = "esc;down;kp_enter;down;down;down;down;kp_enter"
+    default_key = "f9;y"
+    change_config = "kp_enter;down;kp_enter"
```

Comment:
change it to change_config = "kp_enter;down"

Reply:
Ok,many thanks!